modules/timeutil/timestampnano.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package timeutil
+
+import (
+	"time"
+
+	"code.gitea.io/gitea/modules/setting"
+)
+
+// TimeStampNano defines a nano timestamp
+type TimeStampNano int64
```

Comment:
I would suggest to keep this file simple and leave a comment: do not use this anymore ..... it's only for that special case. 😁 

Only the following is enough:

```
type TimeStampNano int64

func TimeStampNanoNow() TimeStampNano {
	return TimeStampNano(time.Now().UnixNano())
}

func (tsn TimeStampNano) AsTime() (tm time.Time) {
	return tsn.AsTimeInLocation(setting.DefaultUILocation)
}
```

Reply:
Done.